### PR TITLE
#184637256 admin able to disable an account

### PR DESCRIPTION
**Pull request by masterchief-00**
#### What does this PR do?

- Adds a feature to allow admin to disable an account

#### Description of Task to be completed?
 Admin Should be able to disable an account
#### How should this be manually tested?

- start the server and then use route /api/v1/api-docs
- create a fake admin user through /api/v1/users/signup
- login through /api/v1/users/signin
- use endpoint /api/v1/users/:id/status to disable/enable an account

#### Any background context you want to provide?
The signup endpoint used here is just for testing not the real signup
#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637256 
#### Screenshots (if appropriate)
 none
#### Questions:
 none

### CODE REVIEW

**Review by dean-daryl — APPROVED** [review]

**Review by rwamugema — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,18 +1,29 @@
+<<<<<<< HEAD
 import * as dotenv from 'dotenv';
 import { app } from './server';
 import db from './database/models/index';
 
+=======
+import * as dotenv from "dotenv";
+>>>>>>> e808b2c (ft(): admin able to disable an account)
```
Comment (fab-ryan) [inline src/index.js]: those line you need to  rebase well

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 (async () => {
   try {
     await db.sequelize
       .sync()
+<<<<<<< HEAD
       .then(() => console.log('-->connected to the db'));
+=======
+      .then(() => console.log("-->connected to the db"));
+>>>>>>> e808b2c (ft(): admin able to disable an account)
   } catch (error) {
```
Comment (fab-ryan) [inline src/index.js]: same as here

File: src/routes/userRoutes.js [hunk 3]
```diff
@@ -17,6 +18,7 @@ userRoutes.get('/api/v1/users', isLoggedIn, getAllUsers);
 userRoutes.post('/api/v1/users/signin', validator(userSchema), loginUser);
 userRoutes.post('/api/v1/users/signup', validator(userSchema), createNewUser);
 userRoutes.put('/api/v1/users/:id/roles', verifyRole('admin'), setRoles);
+userRoutes.put('/api/v1/users/:id/status', verifyRole('admin'), disableAccount);
```
Comment (fab-ryan) [inline src/routes/userRoutes.js]: use patch because you going to update column and change the endpoint `/api/vi/users/:id/disable`

Comment (masterchief-00) [reply src/routes/userRoutes.js]: That endpoint does both enabling and disabling.
when the account is enabled the endpoint disables it and,
when the account is disabled the endpoint enables it. I was thinking maybe the endpoint should be left as it is now

**Review by fab-ryan — APPROVED** [review]
👍

**Pull request merged**